stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/TransformStyle.kt

```diff
@@ -32,6 +33,7 @@ public object TransformStyle {
     public var fileAttachmentStyleTransformer: StyleTransformer<FileAttachmentsViewStyle> = noopTransformer()
     public var suggestionListStyleTransformer: StyleTransformer<SuggestionListViewStyle> = noopTransformer()
     public var messageListHeaderStyleTransformer: StyleTransformer<MessageListHeaderViewStyle> = noopTransformer()
+    public var searchInputViewStyle: StyleTransformer<SearchInputViewStyle> = noopTransformer()
```

Comment:
How about `searchInputViewStyleTransformer` for consistency?

Reply:
Yup, renamed